File: C++/palindrome.cpp
// 8) Program to check if given number is palindrome or not
#include<bits/stdc++.h>
using namespace std;

int main(){
    int num;
    cin>>num;

    int reverse = 0, numcopy = num;
    while(numcopy != 0){
        reverse = reverse * 10 + numcopy % 10;
        numcopy = numcopy / 10;
    }
    if(reverse == num){
        cout<<"Number is Palindrome."<<endl;
    }
    else{
        cout<<"Number is NOT Palindrome."<<endl;
    }
    return 0;
}

File: C#/palindrome.cs
class Program {  
    static void Main(string[] args) {  
        int num, rem, sum = 0, temp;  
        //clrscr();    
        Console.WriteLine("\n >>>> To Find a Number is Palindrome or not <<<< ");  
        Console.Write("\n Enter a number: ");  
        num = Convert.ToInt32(Console.ReadLine());  
        temp = num;  
        while (num > 0) {  
            rem = num % 10;  
            num = num / 10;
            sum = sum * 10 + rem; 
        }  
        Console.WriteLine("\n The Reversed Number is: {0} \n", sum);  
        if (temp == sum)   
        {  
            Console.WriteLine("\n Number is Palindrome \n\n");  
        } else {  
            Console.WriteLine("\n Number is not a palindrome \n\n");  
        }  
        Console.ReadLine();  
    }  
}  
